### DISCUSSION TITLE
Open development model/public working development repo

### DISCUSSION THREAD

**Discussion by cirosantilli** [CovertLab/WholeCellEcoliRelease#23] [Ideas]
As I understand from the README, development happens on another repository, which seems to be private/on request access (I think I had read that somewhere else?)

Just curious, what is the main reason that the project has decided to use a closed development? Is it to maintain confidentiality to not get ripped off by other people publishing faster than you? Or is it to avoid noise from newbs like me? :-) Or another reason?

Code drop makes me sad :cry: (and gives you worse SEO, less trust, duplicates private/public discussions, etc.)

Thanks for this cool project! :rose:

**Comment by tahorst** [comment]
Thanks for your interest in the project!  I would agree that this is a reasonable idea and we have discussed it multiple times as a group.  However, one of the main reasons that it remains a closed repo for development is because we are an academic lab with limited support for an open repo.  We feel a private development repo is best for our needs as we focus on finishing our PhDs and producing papers.  This allows us to release versions of the model to accompany papers once new features are complete and well validated and allows us to appropriately recognize individual contributions to papers when they are published.  We want to make sure we share our work with the community and we are glad to see your interest but for now, we will only have updates associated with publications.
